### DISCUSSION TITLE
updateFromStore - mixin not working from custom component

### DISCUSSION THREAD

**Discussion by venkataadithan** [area17/twill#2435] [Q&A]
## Description

**Update From Store** Mixin hook not working in cutom blocks.

### Steps to reproduce

- create a new input type 
- create a new block and include ckeditor
- get default value from store using "UpdateFromStore" mixin
- default value not fetched from vuexstore.

### Expected result

 updateFromStore hooks/works on custom block component mounting.

### Versions

Twill version:  2.3
Laravel version: 6.x
PHP version: 8.x
Database engine: Mysql

**Comment by ifox** [comment]
For `updateFromStore` to work, you need to make sure your custom field is pushing it's name and value to the vuex store. This is inlined in Blade views for all existing form fields.
